src/redux/reducers/componentConfig/initialState.js

```diff
@@ -296,6 +298,34 @@ const interactiveHeatmapInitialConfig = {
   legendIsVisible: true,
 };
 
+// DATA INTEGRATION - Frequency
+const dataIntegrationFrequencyInitialConfig = {
```

Comment:
many of the fields in `dataIntegrationFrequencyInitialConfig` have values that are the same as the values for `frequencyInitialConfig`. To make this easier to change and more clear, what you can do is make `dataIntegrationFrequencyInitialConfig`  be based on `frequencyInitialConfig` and then just specify the fields that differ. Something like this:

```
const dataIntegrationFrequencyInitialConfig = {
    ...frequencyInitialConfig,
    proportionGrouping: 'sample',
    and so on
}
```